tmt/steps/execute/internal.py

```diff
@@ -20,37 +20,38 @@
     "/usr/local/bin/rstrnt-reboot",
     "/usr/local/bin/rhts-reboot",
     "/usr/local/bin/tmt-reboot")
+REPLACEMENTMARKER = "TMTRestraintForReplacement"
```

Comment:
I don't quite understand what is the advantage of `REPLACEMENTMARKER`. But I assume that if we split the shell scripts from this file to separate files, the logic for inserting information may slightly change anyway

Reply:
As proposed in the meeting a few weeks ago, I based this code off of the already implemented rstrnt-reboot command. That logic made use of three scripts. The 'template' script was the actual code of the rstrnt-reboot command that would reside in /usr/local/bin. The 'setup' script was to simply copy the 'template' script to /usr/local/bin, and the 'teardown' script deleted the script from /usr/local/bin at the end.

The 'setup' and 'teardown' scripts both made explicit reference to the rstrnt-reboot script in /usr/local/bin. By adding the REPLACEMENTMARKER in to the 'setup' and 'teardown' scripts, I have made them generic, and as such reusable for all subsequent rstrnt scripts which are implemented to complete TMT #1027 .

Both the 'setup' and 'teardown' scripts are quite small too, so I think they can remain declared as variables in the internal.py module, whereas the 'template' scripts can exist as separate files.